<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSelectDialog</class>
 <widget class="QDialog" name="ImageSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ImageSelectDialog{
        border-image: url(:/dialog/rc/dialog/dialogbg.png);
}

QLabel{
        font:800 15px/25px &quot;黑体&quot;;
        color: rgba(255,255,255,250);
        border: 2px dashed rgba(180,130,100,0);
        border-radius: 15px;
    	padding: 2px 4px;
}

QListWidget{
        font-weight: 800;
        color: rgba(199,199,14,250);
        border: 2px dashed rgba(180,130,100,0);
        border-image: url(:/Setting/rc/Setting/rclistbg.png);
        border-radius: 10px;
    	padding: 2px 4px;
}
QListWidget::item:selected{ border:none; padding:0px; margin:0px; }
QListWidget::item:selected::!active{ border-width: 0px ; }
QListWidget::item:selected::active{ border-width: 3px; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/*
 水平滚动条
*/
QScrollBar:horizontal {
    background: rgba(0, 0, 0, 0);
    height: 9px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
    background: rgba(195, 195, 195, 255);
    min-width: 20px;
    margin: 2px 0px 1px 0px;
    border-radius: 3px;
    border: 0px dashed rgba(180,130,100,0);
}
QScrollBar::add-line:horizontal {
    border: 0px solid rgba(180,130,100,0);
    background: rgba(180,130,100,0);
    width: 0px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
    border: 0px solid rgba(180,130,100,0);
    background: rgba(180,130,100,0);
    width: 0px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: rgba(180,130,100,0);
}
</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_preview">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Setting.qrc">:/icon/rc/icon/busy.gif</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLineEdit" name="line_imagepath"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_imageselect">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Setting.qrc">
             <normaloff>:/icon/rc/icon/add.png</normaloff>:/icon/rc/icon/add.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Setting.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImageSelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImageSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>

